content/build.js

```diff
@@ -0,0 +1,9 @@
+var fs = require("fs");
+function build() {
+  const getIndustries = fs.readdirSync("./src/roadmaps/industries/").reduce((pv, cv) => {
+    pv.push(require(`./src/roadmaps/industries/${cv}`));
+    return pv;
+  }, []);
+  fs.writeFileSync("./src/industry.json", JSON.stringify({ industries: getIndustries }));
```

Comment:
Can we write this to the lib folder instead of the src folder?  
I think keeping the generated code separate from the actual code is cleaner